src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx

```diff
@@ -0,0 +1,124 @@
+
+<Callout variant="tip">
+If you're using Free or Standard editions, you only have one account available and you won't need this doc. Feel free to revisit this page if you decide to change to [Pro or Enterprise](https://newrelic.com/pricing)
+</Callout>
+
+## Before you get started [#begin]
+
+In this guide we're going to help you create and manage your New Relic account structure. 
+
+First, make sure you're familiar with our terms:
+
+* **Organization**:  An organization represents a New Relic customer. Your New Relic organization contains your organization's accounts, users, and data.
+* **Account**: When you sign up for New Relic your organization is automatically assigned one account. An account can have many sources of data reporting to it. For example, a single account could have data reporting from our infrastructure agent, an APM agent, and other integrations. All data reported to New Relic requires an account ID, which tells New Relic which account that data belongs in. That ID is also used for some account-specific tasks, like making API calls. 
+* **User**: In your organization, your [New Relic users](/docs/accounts/accounts-billing/new-relic-one-user-management/introduction-managing-users/) are granted access to specific accounts that are relevant to their duties and responsibilities. To manage users' access to accounts, you grant groups access to specific roles on specific accounts. Our user management system allows you to create the user access you need, whether that's a relatively simple setup with just a few roles across a few accounts, or a complex one with many roles across many accounts. 
+
+## Single account structure [#single-account-structure]
+
+Signing up for New Relic automatically generates an organization. An organization contains one account unless you manually create more. One account can be perfect for many companies and it might not be necessary for you to add more. 
+
+Some indications that one account is right for you:
+
+* You have 200 employees or less and you don't anticipate growing too much
+* You aren't ingesting vast amounts of data. If you're not sure, use [our spreadsheet](https://newrelic.com/blog/nerdlog/estimate-data-cost) to estimate the amount of data you might send to New Relic
+* You expect to send a moderate amount of alerts. If you're not sure, review our [alerting rules and limits](/docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/rules-limits-alerts/)
+
+If these apply to you, then you might only need one account. Otherwise, we recommend considering a multi-account structure to fit your needs.
+
+## Create multiple accounts [#multiple-accounts]
+
+There are many reasons to create multiple accounts within your organization. Here are some common examples that can help you decide if a multi-account structure is right for you and your team.
+
+### Use multiple accounts to separate testing and production environments [#environments]
```

Comment:
It seems like all software companies would be interested in monitoring each phase of the development lifecycle. Are we suggesting that all Pro and Enterprise edition organizations follow this? What are the benefits and drawbacks?